blockchain/azuredeploy.json

```diff
@@ -100,7 +103,7 @@
     "addressPrefix": "10.0.0.0/16",
     "imagePublisher": "Canonical",
     "imageVersion": "latest",
-    "imageSKU": "14.04.4-LTS",
+    "imageSKU": "18.04.3-LTS",
```

Comment:
this image doesn't exist - you can use 18.04-LTS

Reply:
Fixed.